### ISSUE TITLE
Pylint error in jupyter notebook

### ISSUE THREAD

**Issue by lehoanganh298** [microsoft/vscode-jupyter#6902]
Issue Type: <b>Bug</b>

When using jupyter notebook in VSCode (new UI), Pylint often make wrong warnings in the code not contain any error. Restart Language Server make these linting disappear, but they come back when continue editing.

Extension version: 2021.8.1054968649
VS Code version: Code 1.58.2 (c3f126316369cd610563c75b1b1725e0679adfb3, 2021-07-14T22:10:15.214Z)
OS version: Windows_NT x64 10.0.19042
Restricted Mode: No
Remote OS version: Linux x64 4.19.128-microsoft-standard
Remote OS version: Linux x64 4.19.128-microsoft-standard
Remote OS version: Linux x64 4.19.128-microsoft-standard

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8265U CPU @ 1.60GHz (8 x 1800)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.22GB (1.18GB free)|
|Process Argv|--folder-uri vscode-remote://wsl%2Bubuntu/home/hoanganh/kiki_deep_QA --crash-reporter-id 4f5296a6-ea01-417d-8720-79952247e70a|
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|WSL: Ubuntu|
|OS|Linux x64 4.19.128-microsoft-standard|
|CPUs|Intel(R) Core(TM) i5-8265U CPU @ 1.60GHz (8 x 1800)|
|Memory (System)|11.87GB (7.20GB free)|
|VM|0%|

|Item|Value|
|---|---|
|Remote|WSL: Ubuntu|
|OS|Linux x64 4.19.128-microsoft-standard|
|CPUs|Intel(R) Core(TM) i5-8265U CPU @ 1.60GHz (8 x 1800)|
|Memory (System)|11.87GB (7.20GB free)|
|VM|0%|

|Item|Value|
|---|---|
|Remote|WSL: Ubuntu|
|OS|Linux x64 4.19.128-microsoft-standard|
|CPUs|Intel(R) Core(TM) i5-8265U CPU @ 1.60GHz (8 x 1800)|
|Memory (System)|11.87GB (7.20GB free)|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
pythonvspyt602:30300191
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes627:30244334
pythonvspyt639:30300192
pythontb:30283811
pythonvspyt551:30311712
vspre833:30321513
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
vscus158:30321503
pythonvsuse255:30340121
vscod805cf:30301675
pythonvspyt200:30340761
vscextlang:30333561
binariesv615:30325510
pythonvssor306:30344512
bridge0708:30335490

```

</details>

**Comment by lehoanganh298** [comment]
![image](https://user-images.githubusercontent.com/26211584/127482233-f28668e9-7ad5-4b67-9ffb-a9f77768f893.png)

![image](https://user-images.githubusercontent.com/26211584/127482193-f0baec40-8fd5-4b48-ba2e-dd31ca12a176.png)

**Comment by DonJayamanne** [comment]
Thanks for filling this issue and I'm sorry you're running into this issue.
Please could you provide the version information for the python extension

**Comment by lehoanganh298** [comment]
I'm using:
- Jupyter Extension version v2021.8.1054968649
![image](https://user-images.githubusercontent.com/26211584/127799524-277bae45-2d8a-481b-b8b3-2c1c55f8dad2.png)
- Python kernel: Python 3.8.5 64-bit
- VSCode in Windows 10 with WSL 2
- VS Code Version: 1.58.2 (user setup)
Commit: c3f126316369cd610563c75b1b1725e0679adfb3
Date: 2021-07-14T22:10:15.214Z
Electron: 12.0.13
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Windows_NT x64 10.0.19042

Let me know if you need more information

**Comment by rchiodo** [comment]
This should be fixed in our next version (VS Code 1.6.0)
